### DISCUSSION TITLE
CoPilot Repeats Human Verifcation

### DISCUSSION THREAD

**Discussion by alexchappy** [community/community#118485] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hey! I just used CoPilot (i'm not a code expert) and the human verification kept repeating. Does anyone else have this problem?

**Comment by jfullstackdev** [comment]
you mean in VSCode ? 

you need to log in with the account that is subscribed to Copilot, usually if you try to sign in from VSCode, it will go to Chrome and your GitHub Account that is subscribed to Copilot must be logged in too,

**Reply by alexchappy** [reply]
Actually, no. I used it on bing. I was signed in to my microsoft account.

**Reply by Mohammadkinggh** [reply]
Any fix till now? i mean like wow this problem is like a Plague
